docs/for_linux_developers.md

```diff
@@ -98,15 +98,15 @@ FLASK_SQLALCHEMY_DATABASE_URI="sqlite:///instance/db.sqlite" poetry run ./yaptid
 ```
 
 
-### Testing
+## Testing
 
 To run tests use:
 
 ```shell
 poetry run pytest
 ```
 
-### Development
+## Development
 
 To maintain code quality, we use yapf and flake8. You can run them with. To avoid running them manually we strongly recommend to use pre-commit hooks. To install them run:
```

Comment:
Do we use both yapf and flake8 ?

Reply:
It seems I did not correctly merged latest development section. We are now only using YAPF.

Reply:
Good !